src/managers/oidc-client.ts

```diff
@@ -25,7 +26,7 @@ export const recordNewConnection = async ({
   accountId: string;
   // tricky way to get the non exported Client type
   client: NonNullable<KoaContextWithOIDC["oidc"]["client"]>;
-  params: KoaContextWithOIDC["oidc"]["params"];
+  params: OIDCContextParams;
```

Comment:
I find it weird to use our own overriding of the lib types here

Reply:
Sadly the params you describe is an `UnknownObject`
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6ee5ea358fd4a3eba89662ac7fcf37bf8dff3a94/types/oidc-provider/index.d.ts#L790

That the reason behind the `OIDCContextParams`, an explicit extention of that `UnknownObject`